console/billing/reference-content/reading-an-invoice.mdx

```diff
@@ -0,0 +1,32 @@
+---
+meta:
+  title: Reading an invoice
+  description: This page explains how to read an invoice
+content:
+  h1: Reading an invoice
+  paragraph: This page explains how to read an invoice
+tags: billing invoice payment 
+dates:
+  validation: 2024-02-23
+  posted: 2024-02-23
+categories:
+  - billing
+---
+
+<Lightbox src={["Facture1.webp", "Facture2.webp"]} size={"medium"} alt={["Facture1", "Facture2"]} />
```

Comment:
We have a naming convention for images, they should start with "scaleway", then the naming should indicate what the image is about => "scaleway-read-invoice-one" and "scaleway-read-invoice-two"